### ISSUE TITLE
[Accessibility] Ctrl+8 (Show Environment Pane) does not move screen reader focus

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#12191]
## System Info

RStudio 2022.07.1+554 "Spotted Wakerobin" Release (7872775ebddc40635780ca1ed238934c3345c5de, 2022-07-22) for Ubuntu Bionic
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36

## Reproducible Steps

1. Launch RStudio server.

2. Enable accessibility mode.

3. Load screen reader (e.g., JAWS or NVDA).

4. Press `Ctrl+8`.

* Notes: Whereas `Ctrl+1` and `Ctrl+2` move screen reader focus to each corresponding pane, `Ctrl+8` does not.

**Comment by gtritchie** [comment]
Also covered by this more general issue (#5652) but worth keeping open since fixing these is a pane-by-pane exercise. Or maybe pain.

**Comment by gtritchie** [comment]
The grid control showing the environment values has no keyboard interface and cannot receive keyboard focus. That needs to be addressed!
